edk2toolext/invocables/edk2_pr_eval.py

```diff
@@ -120,6 +120,22 @@ def get_packages_to_build(self, possible_packages: list) -> dict:
         if rc != 0:
             return {}
 
+        #
+        # Policy 0 - A file outside a package was modified and force build
+        #            all packages on a modified file outside a package setting
+        #            is enabled.
+        #
+        # Note: GetContainingPackage() returns the top most directory from the
```

Comment:
this comment is wrong now, right?

Reply:
Yeah, I was waiting for the edk2-pytool-library change to get unblocked to come back to this since it's technically not until the PR over there gets in.

Reply:
Updated